packages/apps/reputation-oracle/server/src/modules/user/user.service.ts

```diff
@@ -369,24 +364,8 @@ export class UserService {
 
   public async registrationInExchangeOracle(
     user: UserEntity,
-    { hCaptchaToken, oracleAddress }: RegistrationInExchangeOracleDto,
+    { oracleAddress }: RegistrationInExchangeOracleDto,
```

Comment:
Maybe change it to
```suggestion
    oracleAddress: string,
```
and pass it as a string in controller?